source/_docs/getting-support.md

```diff
@@ -58,25 +59,26 @@ For more information, see our [Service Level Agreement](https://pantheon.io/sla)
-        <th style="padding-left: 10px;" class="border-top border-bottom border-right">Elite Plan</th>
-    </tr>
+        <th style="padding-left: 10px;" class="border-top border-bottom border-right">Elite Plan/Priority Enterprise Support </th>
+
     <tr>
         <td style="padding-left: 10px;" class="border-bottom border-left border-right">Two 30-minute calls per month with a Senior Customer Success Engineer (scheduled in advance via a support ticket).</td>
         <td style="padding-left: 10px;"class="border-bottom border-right">Unlimited phone support with a Senior Customer Success Engineer (scheduled in advance via a support ticket) and 24/7 emergency support.</td>
+
     </tr>
 </table>
 
 ##FAQs
 
 ####Is there a support number we can call? If so, is this service available for every plan?
 
-For **Elite plans** only, 24/7 emergency support is available post-launch, and we provide an emergency support number to page on duty engineers. For more details, see [Elite Plan Support Overview](https://pantheon.io/resources/pantheon-support-datasheet).
+For **Elite plans** and customers with **Priority Enterprise Support** only, 24/7 emergency support is available post-launch, and we provide an emergency support number to page on duty engineers. For more details, see the [Elite Plan Support Overview](https://pantheon.io/resources/pantheon-support-datasheet) or learn more about [Priority Enterprise Support](https://pantheon.io/priority-enterprise-support).
 
 ####If we open a ticket, do you provide 24/7 support for outages, or are there time restrictions?
 You can file a downtime ticket for outages on our Support page, and we provide 24/7 on-call support for all platform issues affecting paying sites.
 
-####When will you respond after we file a ticket?
-Our guaranteed first response time for Elite plan downtime tickets is 30 minutes and 2 hours for standard tickets. For other paying tiers, we provide best effort resolution time for all tickets on a highest priority, first-come basis.
-
 ####Is 24/7 phone support available for Business plans?
-No, but we do provide two monthly scheduled calls with a Senior Customer Success Engineer for Business plans. Elite plans have access to an emergency line to page the on-call staff 24/7 regarding downtime issues.
+No, but we do provide two monthly scheduled calls with a Senior Customer Success Engineer for Business plans. Elite plans and Priority Enterprise Support customers have access to an emergency line to page the on-call staff 24/7 regarding downtime issues.
+
+#### What is Priority Enterprise Support?
+For organizations that want the highest level of support for their whole team, [Priority Enterprise Support](https://pantheon.io/priority-enterprise-support) provides the assurance of SLA-backed 24/7/365 emergency support, expedited responses to normal tickets, and expert onboarding and training. Contact [Sales](https://pantheon.io/contact-us) for more information.

```

Comment:
This expedited response applies to all tickets pertaining to the organization or member site of the organization filed by any member of the team so long as they are not "unprivileged".